src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs

```diff
@@ -233,8 +212,48 @@ void HandleDirectionChanged(ExpandDirection expandDirection)
 	void SetHeaderGestures(in IView header)
 	{
 		var headerView = (View)header;
-		headerView.GestureRecognizers.Remove(tapGestureRecognizer);
-		headerView.GestureRecognizers.Add(tapGestureRecognizer);
+		headerView.GestureRecognizers.Remove(HeaderTapGestureRecognizer);
+		headerView.GestureRecognizers.Add(HeaderTapGestureRecognizer);
+	}
+
+	void OnHeaderTapGestureRecognizerTapped(object? sender, TappedEventArgs tappedEventArgs)
+	{
+		IsExpanded = !IsExpanded;
+		HandleHeaderTapped?.Invoke(tappedEventArgs);
+	}
+
+	void ResizeExpanderInItemsView(TappedEventArgs tappedEventArgs)
+	{
+		if (Header is null)
+		{
+			return;
+		}
+
+		Element element = this;
+		var size = IsExpanded
+					? Measure(double.PositiveInfinity, double.PositiveInfinity, MeasureFlags.IncludeMargins).Request
+					: Header.Measure(double.PositiveInfinity, double.PositiveInfinity);
+
+		while (element is not null)
+		{
+			if (element.Parent is ListView && element is Cell cell)
```

Comment:
@VladislavAntonyuk you mentioned [here](https://github.com/CommunityToolkit/Maui/pull/751#issuecomment-1380883728) this PR just fix for `CollectionView`, so why do we have a `ListView` handle here?

Reply:
because for listview we still need to throw an unsupported exception on Apple.
But on other platforms, we can use it with ForceUpdateSize().